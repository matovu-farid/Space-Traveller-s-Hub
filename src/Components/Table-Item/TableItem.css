.table-item {
  border: 1px solid gray;
  width: 250px;
  padding: 5px;
}