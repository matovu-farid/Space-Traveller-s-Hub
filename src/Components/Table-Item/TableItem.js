import './TableItem.css';
import propTypes from 'prop-types';

const TableItem = ({ name }) => (
  <li className="table-item">
    {name}
  </li>
);

export default TableItem;

TableItem.propTypes = {
  name: propTypes.string.isRequired,
};
