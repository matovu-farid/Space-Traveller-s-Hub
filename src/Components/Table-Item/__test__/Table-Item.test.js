import React from 'react';
import renderer from 'react-test-renderer';
import TableItem from '../TableItem';

it('renders correctly', () => {
  const tree = renderer
    .create(
      <TableItem name="Rocket" />,

    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});
