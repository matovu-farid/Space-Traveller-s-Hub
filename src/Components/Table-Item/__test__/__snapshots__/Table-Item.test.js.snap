// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<li
  className="table-item"
>
  Rocket
</li>
`;
